examples/intervention_approvals/approvals.py

```diff
@@ -0,0 +1,335 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import Any, Dict, List, Optional, Tuple
+from rich.panel import Panel
+from rich.console import Console, Group, RenderableType
+from rich.markdown import Markdown
+from rich.syntax import Syntax
+from rich.prompt import Confirm, Prompt
+from inspect_ai.util import input_screen
+from inspect_ai.solver._task_state import sample_state
+from inspect_ai.solver import TaskState
+from inspect_ai.model import ModelOutput
+from inspect_ai.tool import ToolCall
+import sys
+import shlex
+
+class ApprovalDecision(Enum):
```

Comment:
We have been using `Literal["approve", "reject", "escalate", "terminate"]` for enums